packages/apps/human-app/frontend/src/smart-contracts/get-network-name.ts

```diff
@@ -1,9 +1,9 @@
-import { MainnetChains, TestnetChains } from '@/smart-contracts/chains';
+import { AllTestnetsChains, AllMainnetChains } from '@/smart-contracts/chains';
 import { env } from '@/shared/env';
 
-export const getNetworkName = (): string => {
+export const getNetworkName = (chainId: number): string | null => {
```

Comment:
Isn't return type always `string`?
In both scenarios you have default `''`

Reply:
That's right, I need to pay more attention when refactoring